lib/modules/manager/helmsman/extract.ts

```diff
@@ -31,6 +35,30 @@ function createDep(
   }
   dep.currentValue = anApp.version;
 
+  // in case of OCI repository, we need a PackageDependency with a DockerDatasource and a packageName
+  const ociRegexResult = anApp.chart ? ociChartRegex.exec(anApp.chart) : null;
+  if (ociRegexResult?.groups) {
+    if (!is.nonEmptyString(ociRegexResult.groups.packageName)) {
+      dep.skipReason = 'invalid-name';
+      return dep;
+    }
+
+    if (!is.nonEmptyString(ociRegexResult.groups.registryRef)) {
+      dep.skipReason = 'no-repository';
+      return dep;
+    }
+
+    dep.datasource = DockerDatasource.id;
+    const ociRegistryUrl = ociRegexResult.groups.registryRef.replace(
+      'oci://',
+      ''
+    );
+    dep.registryUrls = [ociRegistryUrl];
+    dep.packageName = ociRegexResult.input.replace('oci://', '');
```

Comment:
```suggestion
    dep.packageName = anApp.chart.replace('oci://', '');
```
we know it's not null.